.infrastructure/mysql/mysql-secret.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql-secrets
```

Comment:
The secret name `mysql-secrets` should be corrected to `mysql-secret` to match the actual secret name used in your deployment configuration.